packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -36,6 +39,11 @@ const styles: ComponentStyles<IComponentWrapperManagedClasses, IDevSiteDesignSys
             borderLeft: componentWrapperBorder
         },
     },
+    componentWrapperFullWidth: {
+        display: "block",
+        padding: toPx(24),
+        borderBottom: componentWrapperBorder,
+    },
```

Comment:
`componentWrapperFullWidth` looks to contain the base styles needed in componentWrapper as well.

I would consider doing one of the following:
- Instead of passing a completely different class (componentWrapperFullWidth), you can pass an additional class that adds the border modifiers.
- OR, create a function that returns these styles such as `applyComponentWrapperBase() {}`

Reply:
Using either of the suggestions above will also allow us to remove the const for `componentWrapperBorder` on line 29 and just pass it in one place.